include/he.h

```diff
@@ -402,6 +404,18 @@ typedef he_return_code_t (*he_nudge_time_cb_t)(he_conn_t *conn, int timeout, voi
 typedef bool (*he_auth_cb_t)(he_conn_t *conn, char const *username, char const *password,
                              void *context);
 
+/**
+ * @brief The prototype for the authentication token callback
+ * @param conn A pointer to the connection that triggered this callback
+ * @param token A pointer to buffer containing the auth token
+ * @param len Length of the token
```

Comment:
Should we specify the unit of `Length of the token in bytes` here?  I assume it's bytes. Without specifying it, I am wondering if it could mean "number of digits" or something similar.

Reply:
Will do. Btw, `public/he.h` is generated, so we only need to fix the `include/he.h`.